src/storage/indexed-db-adapter.test.ts

```
@@ -0,0 +1,301 @@
+        'IndexedDBAdapter can only be used in browser environments with IndexedDB support'
+      );
+    });
+
+    it('should not throw error if IndexedDB is available', () => {
+      expect(() => adapter['verifyEnvironment']()).not.toThrow();
+    });
+  });
+
+  describe('initDB', () => {
+    it('should initialize database successfully', async () => {
+      const initPromise = adapter['initDB']();
```

ESLint

[error] ["initDB"] is better written in dot notation.

(dot-notation)